src/app.js

```diff
@@ -1,5 +1,206 @@
 const express = require('express');
 
+const { sayHello } = require('./lib/strings');
+const { firstCharacters } = require('./lib/strings');
+const { uppercase } = require('./lib/strings');
+const { lowercase } = require('./lib/strings');
+const { firstCharacter } = require('./lib/strings');
+
+const { add, subtract, divide, multiply, remainder } = require('./lib/numbers');
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
+// Middleware
+
 const app = express();
+app.use(express.json());
+
+// Strings Section
+
+app.get('/strings/hello/world', (_, res) => {
+  res.status(200).json({ result: 'Hello, world!' });
+});
+
+app.get('/strings/hello/:string', (req, res) => {
+  res.status(200).json({ result: sayHello(req.params.string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  // const result = req.params.string.toUpperCase();
+  res.status(200).json({ result: uppercase(req.params.string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.status(200).json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-character/:string', (req, res) => {
+  res.status(200).json({ result: firstCharacter(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  const n = req.query.length;
+  res.status(200).json({ result: firstCharacters(req.params.string, n) });
+});
+
+// Numbers Section
+
+app.get('/numbers/add/:a/and/:b', (req, res) => {
+  const a = parseInt(req.params.a, 10);
+  const b = parseInt(req.params.b, 10);
+  if (isNaN(a) || isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  }
+  res.status(200).json({ result: add(a, b) });
+});
+
+app.get('/numbers/subtract/:a/from/:b', (req, res) => {
+  const a = parseInt(req.params.a, 10);
+  const b = parseInt(req.params.b, 10);
+  if (isNaN(a) || isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  }
+  res.status(200).json({ result: subtract(b, a) });
+  console.log({ result: subtract(b, a) });
```

Comment:
Theres a ton of console logs littered throughout the project, they are really cluttering up the output of the tests! Should definitely remove all of these throughout